gobblin-yarn/src/main/java/gobblin/yarn/YarnService.java

```diff
@@ -12,10 +12,15 @@
 
 package gobblin.yarn;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
The order of imports is wrong. Check out https://github.com/linkedin/gobblin/wiki/CodingStyle.